vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -120,6 +175,7 @@
 		$(window).on("load", function() {
 		    let spaceHeight = $("#bgImage").css("height");
 		    $("#Module_1").css("height",spaceHeight);
+		    $("#Module_1").css("margin", "-45px 0px 60px auto");
```

Comment:
This makes the background image move up after loading; it should be set correctly right away (no jumping up effect).